### PR TITLE
first-setup

### PR DESCRIPTION
**Pull request by pPitohu**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-export default () => <div className="App">EAZY LEARN</div>;
+const App: React.FC = function () {
```
Comment (psychobug420) [inline src/App.tsx]: I would prefer function declaration or using arrow function, I don't understand why you used function expression, you can try to explain me your choice

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
```
Comment (psychobug420) [inline .vscode/settings.json]: I would prefer Prettier than eslint to format code, and it wasn't part of ticket to setup formatters, next time ask me about such questions if ticket too short

File: .eslintrc.json [hunk 3]
```diff
@@ -1,17 +1,48 @@
 {
-  "env": {
```
Comment (psychobug420) [inline .eslintrc.json:2]: This file is okay, in future we would improve eslint rules more

File: src/index.tsx [hunk 4]
```diff
@@ -3,11 +3,11 @@ import ReactDOM from 'react-dom';
 
 import App from './App';
 
-const funct = (): any => "hello"
+const funct = (): any => 'hello';
```
Comment (psychobug420) [inline src/index.tsx]: Please remove this function I forget to remove it, it was just to check eslint

File: src/App.tsx (hunk 1 above)
Comment (pPitohu) [reply src/App.tsx]: Whoops, miss that, i was thinking i'm writing an arrow function

File: .gitignore [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+/.vscode
```
Comment (psychobug420) [inline .gitignore]: remove /.vscode from .gitingore every developers in our team use it)

File: .husky/pre-commit [hunk 6]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (psychobug420) [inline .husky/pre-commit:1]: That file is also okay, but in future we would improve this

**Review by psychobug420 — CHANGES_REQUESTED** [review]

**Pull request merged**